tests/test_level.py

```
@@ -0,0 +1,6 @@
+from femtologging import FemtoLevel
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)